feat: Database Monitoring PostgreSQL documentation [Feb Release] (#19932)
newrelic/docs-website: src/install/postgresql/windows/install-windows-query-monitoring.mdx

> ---

Reviewer: See comments in previous files

headingText: Enable query monitoring
componentType: default
---

> You can view query-level analysis to monitor your SQL queries within your PostgreSQL database to assess their performance and impact. This feature provides insights into execution times, resource consumption, and potential bottlenecks, enabling you to optimize database operations. For more information, see [Query-Level Analysis](#).

Reviewer: ```suggestion
You can view query-level analysis to assess the performance and impact of your SQL queries in your PostgreSQL database. This feature provides insights into execution times, resource consumption, and potential bottlenecks, enabling you to optimize database operations. For more information, see [Query-Level Analysis](#).
```

**Prerequisites:**

> * Ensure that your PostgreSQL version is 12 or higher.

Reviewer: ```suggestion
* PostgreSQL version is 12 or later.
```
